### PR TITLE
Internal: Added missing google maps and text path E2E tests [ED-11076]

### PR DESCRIPTION
**Pull request by dmaziar**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x ] Other... Please describe: Playwright tests

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: tests/playwright/pages/elementor-panel-tabs/content.js [hunk 1]
```diff
@@ -41,11 +41,13 @@ export default class Content {
 		await this.editorPage.getPreviewFrame().locator( EditorSelectors.siteTitle ).click();
 	}
 
-	async verifyLink( element, options = { target, href, rel, customAttributes } ) {
+	async verifyLink( element, options = { target, href, rel, customAttributes, widget } ) {
```
Comment (yotamselementor) [inline tests/playwright/pages/elementor-panel-tabs/content.js:44]: Please add a JSDoc comment specifying the parameters and their types, this is getting confusing

File: tests/playwright/pages/elementor-panel-tabs/content.js [hunk 2]
```diff
@@ -125,4 +130,23 @@ export default class Content {
 		await textEditor.getByText( 'Tab Content' ).click();
 		await textEditor.locator( EditorSelectors.tabs.body ).fill( text );
 	}
+
+	async parseSrc( src ) {
```
Comment (yotamselementor) [inline tests/playwright/pages/elementor-panel-tabs/content.js:134]: What does this do?

**Review by yotamselementor — CHANGES_REQUESTED** [review]

File: tests/playwright/pages/elementor-panel-tabs/content.js (hunk 1 above)
Comment (dmaziar) [reply tests/playwright/pages/elementor-panel-tabs/content.js:54]: `<a href="http://www.link.test" target="_blank" rel="nofollow" mykey="testValue">Add Your Heading Text Here</a>`
`options = { target, href, rel, customAttributes, widget }`
If you look at `a` tag, this is just a standard HTML attribute of this tag. A widget is just a widget name. I don't know how to name it more understandable. 

I have added jsdoc, but this is an extra 10 lines of comments.

File: tests/playwright/pages/elementor-panel-tabs/content.js (hunk 2 above)
Comment (dmaziar) [reply tests/playwright/pages/elementor-panel-tabs/content.js]: Added jsdoc. This one parses a[src] attribute and gets query params and its values. So we can test that our controls set the correct params.

**Review by yotamselementor — APPROVED** [review]

**Pull request merged**